Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -117,17 +154,54 @@
 // Temperature Sensors
 //
 #define TEMP_0_PIN          1   // Analog Input (Extruder)
+#define TEMP_A_PIN          2   // Analog Input (Extrudrboard A THERM)
+#define TEMP_B_PIN          3   // Analog Input (Extrudrboard B THERM)
 #define TEMP_BED_PIN        0   // Analog Input (Bed)
 
 //
 // Heaters / Fans
 //
 #define HEATER_0_PIN       15   // C5 PWM3B - Extruder
-#define HEATER_1_PIN       44   // F6
-#define HEATER_2_PIN       45   // F7
+// Note that Printrboard REV F6 removed HEATER_A_PIN from EXP1 and
+// reassigned it for a second fan output.
+#define HEATER_A_PIN       44   // F6 - Extrudrboard A HOTEND
+#define HEATER_B_PIN       45   // F7 - Extrudrboard B HOTEND
```

Comment:
Same deal here. Regardless of what the board silkscreen reads, the pins must be named according to Marlin conventions.